_data/food.yml

```diff
@@ -31,6 +31,34 @@ websites:
       - "no"
       - se
 
+  - name: Foodora (Finland)
+    url: https://www.foodora.fi/
+    img: foodora.png
+    facebook: foodora.fi
+    email_address: info@foodora.fi
+    regions:
+    - fi
+    lang: fi
+
+  - name: Foodora (Norway)
+    url: https://www.foodora.no/
+    img: foodora.png
+    facebook: foodora.no
```

Comment:
```suggestion
    facebook: foodora.no
    twitter: foodora_no
```

Reply:
Doesn't seem to be monitored. Probably just holding on to the username